admin/app/components/complementary-certifications/attach-badges/badges/index.js

```diff
@@ -22,12 +22,12 @@ export default class Badges extends Component {
   }
 
   @action
-  onBadgeUpdated(badgeId, badgeLevel) {
+  onBadgeUpdated({ badgeId, fieldName, fieldValue }) {
     if (!badgeId) {
       return;
     }
 
-    this.args.onBadgeUpdated({ badgeId, badgeLevel });
+    this.args.onBadgeUpdated({ badges: this.badges, update: { badgeId, fieldName, fieldValue } });
```

Comment:
Il semble que `badges` ne soit pas utilisé
```suggestion
    this.args.onBadgeUpdated({ update: { badgeId, fieldName, fieldValue } });
```

Reply:
Modifié